client/lib/purchases/index.js

```diff
@@ -204,6 +212,19 @@ function hasCreditCardData( purchase ) {
 	return Boolean( purchase.payment.creditCard.expiryMoment );
 }
 
+/**
+ * Checks whether the purchase is capable of explicit renew.
```

Comment:
Good call. It's more than just user interaction. Even "system action" cannot renew these domains -- they _have_ to fall to autorenew (or not).  I'll tweak to make all fully clear as I can.